hack/test-extended.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+source ${OS_ROOT}/hack/util.sh
+source ${OS_ROOT}/hack/common.sh
+
+# TODO: Randomize these ports
+export OS_EXTENDED_MASTER=127.0.0.1:54199
+export OS_EXTENDED_ASSETS=127.0.0.1:54299
+
+export BASETMPDIR="/tmp/openshift-extended"
```

Comment:
Using ${TMPDIR:-/tmp} instead of /tmp would be useful in case you want to use a different temp directory